morallito/IoT-service-provider
# Scaffolding/server

## Add a docker file to server scaffolding

Finish the server scaffolding, add a docker file to run  the project in production.


## How to test it:

> Test the server 

```
cd server
dotnet fake build -t run
```

> Test docker

```
cd server
sudo docker build -t safe-template . 
sudo  docker run -p 8080:8080 safe-template
```

Go to localhost:8080 and see the welcome page,
